### PR TITLE
fix: remove test case that is not required for ExportCommand

### PR DESCRIPTION
**Pull request by danielleloh**

### CODE REVIEW

**Comment by Pughal77** [comment]
Nice! fixes an important issue that arises with our tests

File: src/test/java/seedu/address/logic/commands/ExportCommandTest.java [hunk 1]
```diff
@@ -74,13 +74,6 @@ public void execute_unableToCreateCsvDirectory_throwsCommandException() throws E
         assertEquals("Could not create directory for CSV file.", thrown.getMessage());
     }
 
-    @Test
-    public void execute_withoutValidJsonFilePath_throwsCommandException() {
-        CommandException thrown = assertThrows(CommandException.class, () -> exportCommand
-                .execute(modelWithNoFilePath));
-        assertEquals("Cannot find JSON file to export.", thrown.getMessage());
-    }
-
```
Comment (Zer0Legion) [inline src/test/java/seedu/address/logic/commands/ExportCommandTest.java:83]: good catch! this case is not handled by export class and so we don't need to test it.

**Review by Pughal77 — COMMENTED** [review]
LGTM! it fixes an important issue in our tests

File: src/test/java/seedu/address/logic/commands/ExportCommandTest.java [hunk 2]
```diff
@@ -74,13 +74,6 @@ public void execute_unableToCreateCsvDirectory_throwsCommandException() throws E
         assertEquals("Could not create directory for CSV file.", thrown.getMessage());
     }
 
-    @Test
```
Comment (jayllo-c) [inline src/test/java/seedu/address/logic/commands/ExportCommandTest.java:77]: Great catch figuring out that the test case would not affect code coverage due to the way that the user prefs are created as well as the functions to change the addressbook address!

**Review by jayllo-c — COMMENTED** [review]
Good job simplifying the code coverage!

**Pull request merged**